src/components/summaryDetail/Categories.jsx

```diff
@@ -0,0 +1,14 @@
+import { nanoid } from 'nanoid';
```

Comment:
this import is unnecessary as the logic does not use nanoid.

Reply:
Addressed with commit [a118fed](https://github.com/chingu-voyages/v46-tier2-team-20/pull/51/commits/a118fed58e19eaa879bff38aa690a26fb0bbcc39)